web/src/main/java/org/akaza/openclinica/controller/BatchCRFMigrationController.java

```diff
@@ -125,7 +126,8 @@ public BatchCRFMigrationController(HelperObject helperObject) {
     public void getLogFile(@PathVariable("filename") String fileName, HttpServletResponse response) throws Exception {
         InputStream inputStream = null;
         try {
-            String logFileName = getFilePath() + File.separator + fileName;
+        	//Validate/Sanitize user input filename using a standard library, prevent from path traversal 
+            String logFileName = getFilePath() + File.separator + FilenameUtils.getName(fileName);
```

Comment:
same comment as above

Reply:
changed--check for canonical path  and only allow if the canonical path starts with the base dir path.